src/components/misc/StaticMap.jsx

```diff
@@ -9,6 +10,7 @@ export default class StaticMap extends React.Component {
         location: React.PropTypes.object.isRequired,
         width: React.PropTypes.number,
         height: React.PropTypes.number,
+        placeholder: React.PropTypes.bool
```

Comment:
This should be removed.

If we decide to externalize the strings `setPositionLink` and `noCoordinates` as described below, the `placeholder` property might instead be used for one of them, along the same lines as the HTML5 `placeholder` attribute on inputs.